backend/graph/api/resolver.go

```diff
@@ -480,10 +472,12 @@ func (r *Resolver) sendMessage(ctx context.Context, itemID uuid.UUID, message *s
 		insertParams.AgeGroup = usr.AgeGroup
 	}
 
-	err = r.Queries.SetMessage(ctx, insertParams)
-	if err != nil {
-		log.L.Error().Err(err).Msg("Failed to save string to database")
-		return "", merry.New("Failed to generate unique ID")
+	if !user.IsImpersonating(gc) {
+		err = r.Queries.SetMessage(ctx, insertParams)
+		if err != nil {
+			log.L.Error().Err(err).Msg("Failed to save string to database")
+			return "", merry.New("Failed to generate unique ID")
+		}
```

Comment:
Why are we not doing it here? I feel that this mutations are an important part of the load test...

Reply:
It's important yes, so all the "impersonating" things can probably be changed or ignored, but I don't want to write unnecessarily or by mistake to the database until we have a proper staging or testing environment.

I don't think I will be doing any mutations until I've made the fetch requests, and when we need to test writes, we can easily just remove these checks.